.github/workflows/main.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           go-version: 1.17
 
       - name: Build
-        run: go build -v ./...
+        run: |
+          git_hash=$(git rev-parse --short "$GITHUB_SHA")
```

Comment:
@PankhudiB as discussed in the issue thread. We should have both the commit-sha and the git-tag logged for the version. We mostly use git-tag for versions but adding commit-sha can. help identify unreleased builds.